src/pke/lib/scheme/bfvrns/bfvrns-parametergeneration.cpp

```diff
@@ -138,7 +138,7 @@ bool ParameterGenerationBFVRNS::ParamsGenBFVRNS(std::shared_ptr<CryptoParameters
             // conservative estimate for HYBRID to avoid the use of method of
             // iterative approximations; we do not know the number
             // of digits and moduli at this point and use upper bounds
-            double numTowers = ceil(static_cast<double>(logqPrev) / dcrtBits);
+            double numTowers = ceil(logqPrev / dcrtBits);
```

Comment:
std::ceil()

Reply:
missed this one. i guess we can leave it as is. i am sure that we will return to this code in the nearest future to fix this and some other things in the file